libs/types-common/src/Data/Json/Util.hs

```diff
@@ -212,9 +211,6 @@ base64SchemaN = toBase64Text .= parsedText "Base64ByteString" fromBase64Text
 base64Schema :: ValueSchema SwaggerDoc ByteString
 base64Schema = unnamed base64SchemaN
 
-toBase64ByteString :: ByteString -> Base64ByteString
```

Comment:
So apparently it was being used :)

Reply:
Oh, my bad. I used it in a test, but now we know it was a wrong thing to do. I replaced its usage with the `Base64ByteString` data constructor.